resource/closecaption_german.txt

```diff
@@ -970,65 +970,65 @@
 "[english]crash.WatchOut1"		"<clr:238,180,34>Crash: There!"
 "crash.WatchOut2"		"<clr:238,180,34>Crash: Pass auf!"
 "[english]crash.WatchOut2"		"<clr:238,180,34>Crash: Watch out!"
-"crash.Shieldbug0"		"<clr:238,180,34>Crash: Heiliger Strohsack!"
+"crash.Shieldbug0"		"<clr:238,180,34>Crash: Heilige Scheiße!"
```

Comment:
Ja, das ist gut. Wir haben sogar explizite Hinweise in der [Spielbeschreibung](https://github.com/ReactiveDrop/reactivedrop_translations/blob/master/store_page/content_warning_563560_german.txt) das an einigen Stellen ein bisschen geflucht wird.